cli/setup.py

```diff
@@ -77,6 +77,7 @@ def readme():
     entry_points={
         "console_scripts": [
             "pcluster = pcluster.cli.entrypoint:main",
+            "pcluster3-config-converter = pcluster.cli.commands.configuration_converter:main",
```

Comment:
if possible I'd move `pcluster3-config-converter` outside of the pcluster dir to something like cli/src/pcluster3_config_converter and avoid any dependency with the modules defined in pcluster package if possible

Reply:
Moved it outside